pkg/cmd/cli/cmd/buildlogs.go

```diff
@@ -16,7 +16,7 @@ func NewCmdBuildLogs(f *Factory, out io.Writer) *cobra.Command {
 NOTE: This command may be moved in the future.
 
 Examples:
-$ kubectl build-logs 566bed879d2d
```

Comment:
A space between the script and `Examples:` and 4 instead of 2 spaces of indentation is needed here in order for the markdown to work. See [here](https://github.com/GoogleCloudPlatform/kubernetes/pull/4065) and [here](https://github.com/openshift/origin/pull/794#discussion_r24085631)